simple-no-auth/src/test/java/com/fga/example/OpenFgaContainerConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.fga.example;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+
+@TestConfiguration(proxyBeanMethods = false)
+class OpenFgaContainerConfiguration {
+
+	@Bean
+	GenericContainer<?> openFgaContainer(DynamicPropertyRegistry registry) {
+		var result = new GenericContainer<>("openfga/openfga:latest")
```

Comment:
You may want to add `.withReuse(true)` for those who have it enabled (e.g. Testcontainers Desktop users have it enabled by default), to shorten the iteration cycles 👍

Reply:
Thanks for this pointer. I'm not going to do this since it is marked as experimental and we aren't having performance issues at the moment.